src/pages/Settings.vue

```diff
@@ -54,6 +65,14 @@
 <script>
 import { useRoute } from "vue-router";
 
+/** Deduplicate an array of objects using the provided key
+* @param {Object[]} arr array of objects to deduplicate
+* @param {string} key key used for uniqness
+* @returns {Object[]} the deduplicated array
+*/
+const uniqBy = (arr, key) =>
```

Comment:
I think inlining this function might be a okay (this fucntion seems sort of weird)

Reply:
Not sure the well understand your point? 🤔 
I don't copy/past it from some obscur stackoverflow thread, I write it myself, using modern vanilla JS features (Map).
Why did you thing it as `sort of weird`? 😮 

Also, I extract it because it's a very generic helper and it make things very clearer at L.152:
```js
const settingRoutes = uniqBy(
                $route.matched.filter(({ path }) => /^\/settings\//.test(path)),
                "path"
            );
```

Reply:
Let's merge this with https://github.com/M1CK431/uptime-kuma/pull/1/files/f306f5aa50f46a474e0f058c82c8701520804e9e#r1287775085